src/Route.jsx

```diff
@@ -13,10 +17,27 @@ const STOP_ICON_MAPPING = {
 
 const BUS_ICON_MAPPING = {
   marker: {
-    x: 0, y: 0, width: 128, height: 128, mask: false,
+    x: 0, y: 0, width: 128, height: 128, mask: true,
```

Comment:
Any reason we're changing this?

Reply:
I change from false to true because it allow the bus icon to change color base on different bus lines